### PR TITLE
[WIP] frontend redesign

### PR DESCRIPTION
**Pull request by bchieng**
This doesn't currently work because it hasn't been rebased against master in a long time but should be enough as a proof of concept. Screenshot attached of the web redesign running on a port before it broke with recent ocflib changes. I'll work on doing this again (with @lamalor's code + help) from a fresh checkout of master so it can be built. TY to @abizer @keur  for assistance during the process 

46276201_285168191998277_8667791607363272704_n

### CODE REVIEW

File: ocfweb/main/templates/main/home.html [hunk 1]
```diff
@@ -3,19 +3,28 @@
...
+                <h1>Welcome to the Open Computing Facility!</h1><br>
                 <p>The Open Computing Facility (OCF) is an all-volunteer student organization dedicated to free computing for all University of California, Berkeley students, faculty, and staff.</p>
-                <p class="ocf-button-holder">
-                <a href="{% url 'about-staff' %}" class="btn btn-lg">Learn what we do</a>
-                <a href="{% url 'register' %}" class="btn btn-lg">Create an account</a>
+            </div>
+            <div class="col-sm-6 ocf-login-main">
+                <a class="sso-button" title="For current students only" id="sso-button" data-bind="click: track_sso_click" href="https://auth.berkeley.edu/cas/login?service=https%3A%2F%2Fwww.ocf.berkeley.edu%2Flogin%2Fcalnet%2Flogin%2F%3Fnext%3D%252Faccount%252Fregister%252F&renew=true">
+                  <div class="sso-content">
+                    <img alt="" src="https://s3.amazonaws.com/handshake.production/app/public/assets/schools/84/small/hs-school-logo-data.?1482461526">
```
Comment (dkess) [inline ocfweb/main/templates/main/home.html]: This image should be hosted by us, from static assets, not from another website.

File: ocfweb/main/templates/main/home.html [hunk 2]
```diff
@@ -3,19 +3,28 @@
...
-                <h1>Welcome to the Open Computing Facility!</h1>
+            <div class="col-sm-6">
+                <h1>Welcome to the Open Computing Facility!</h1><br>
                 <p>The Open Computing Facility (OCF) is an all-volunteer student organization dedicated to free computing for all University of California, Berkeley students, faculty, and staff.</p>
-                <p class="ocf-button-holder">
-                <a href="{% url 'about-staff' %}" class="btn btn-lg">Learn what we do</a>
-                <a href="{% url 'register' %}" class="btn btn-lg">Create an account</a>
+            </div>
+            <div class="col-sm-6 ocf-login-main">
+                <a class="sso-button" title="For current students only" id="sso-button" data-bind="click: track_sso_click" href="https://auth.berkeley.edu/cas/login?service=https%3A%2F%2Fwww.ocf.berkeley.edu%2Flogin%2Fcalnet%2Flogin%2F%3Fnext%3D%252Faccount%252Fregister%252F&renew=true">
```
Comment (dkess) [inline ocfweb/main/templates/main/home.html]: Is `data-bind` used here? I suspect this is from Handshake, we can probably get rid of it.

File: ocfweb/static/scss/site.scss [hunk 3]
```diff
@@ -73,9 +74,9 @@ h6 {
 }
 
 .ocf-navbar {
-    @extend .navbar-default;
 
-    background-color: rgba(255, 255, 255, 0.9);
+    // baby blue with opacity changed (http://hex2rgba.devoth.com/)
+    background-color: rgba(203, 225, 234, 1.0);
```
Comment (dkess) [inline ocfweb/static/scss/site.scss]: If opacity is 1, it doesn't have to be specified at all

File: ocfweb/static/scss/site.scss [hunk 4]
```diff
@@ -211,31 +217,40 @@ h6 {
...
+            margin-top: 5px;
+            font-size: 20px;
+        }
+    }
 }
 
 .ocf-content-block {
     > :first-child {
-        margin-top: 20px;
+        margin-top: 0px;
```
Comment (dkess) [inline ocfweb/static/scss/site.scss]: For 0 values you don't need to specify units

File: ocfweb/static/scss/site.scss [hunk 5]
```diff
@@ -211,31 +217,40 @@ h6 {
...
-.ocf-sidebar {
-    @extend .ocf-content-block;
-}
 
 .ocf-hero {
     background-color: $hero-color;
     font-size: 16px;
-    padding: 0 10px;
+    text-align: center;
+    padding: 15px 15px 15px 15px;
```
Comment (dkess) [inline ocfweb/static/scss/site.scss]: `padding: 15px`

File: ocfweb/about/templates/about/staff.html [hunk 6]
```diff
@@ -12,7 +12,7 @@
                 <div class="col-sm-10">
                     <img style="float: left; margin-right: 20px; margin-bottom: 10px;" class="visible-xs" src="{% static 'img/penguin.svg' %}" width="90" />
                     <h1>What is the Open Computing Facility?</h1>
-                    <p>We're a student organization dedicated to providing free computing for all UC Berkeley students, faculty, and staff.</p>
+                    <p> We're a student organization dedicated to providing free computing for all UC Berkeley students, faculty, and staff.</p>
```
Comment (TonyLianLong) [inline ocfweb/about/templates/about/staff.html]: Why do we need a space here?

**Comment by nikhiljha** [comment]
I think @bchieng has graduated and some current staffers are working on https://github.com/ocf/ocfstatic - so closing this.

**Pull request closed without merging**